### ISSUE TITLE
[Performance] Can we set model weight precision when converting keras model into onnx model?

### ISSUE THREAD

**Issue by tjk9501** [microsoft/onnxruntime#15695]
### Describe the issue

Hello!

I train a CNN model with tensorflow, and save it into model.best(1).pb folder containing two *.pb files using 
```
tensorflow.keras.models.save_model().
```
 After that, I convert the CNN model into *.onnx format using the following command
```
python -m tf2onnx.convert --saved-model './model.best(1).pb' --output ./model.onnx --opset 11 --verbose
```

However, when I test the saving onnx model in Visual Studio 2022 C++, it turns out the accuracy of predicting results degraded compared with using *.hdf5 keras model in tensorflow and python. I have seen some people say the predicting accuracy of model decrease because the decimal precision of model weight parameters decreases during conversion with onnx runtime. Could anyone offer a method which can effectively control the model weight parameter precision during onnx runtime conversion?

Thanks!

### To reproduce

Platform: python 3.10 with tensorflow 2.5
model format in tensorflow : keras hdf5 file
model conversion: first to pb folder and then to onnx
conversion method:
```
python -m tf2onnx.convert --saved-model './model.best(1).pb' --output ./model.onnx --opset 11 --verbose
```
Input tensor shape : [None, 1024,1]
Output tensor shape: [None,34375]

Possible to test with random numbers

### Platform

Windows

### OS Version

Microsoft windows 10 or windows 11

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.14.1 with Microsoft Visual Studio C++ 2022

### ONNX Runtime API

Python

### Architecture

X64

### Execution Provider

Default CPU

### Is this a quantized model?

Yes

**Comment by tjk9501** [comment]
Hello!

I think maybe I didn't express the problem very clearly. The original keras model is about 111 MB, after converting to onnx format, the model size decreases to 37 MB. I want to convert to an onnx model with its size still retain as 111 MB, as I suspect when compressing to an 37 MB onnx model, parameter precision was lost.
